docs/development.rst

```diff
@@ -445,6 +448,86 @@ finally force-push to the remote topic branch on your fork::
+  first author, and the year the study was published). For example, the Gutenkunst et al.
+  (2009) Out of Africa demographic model has the ``id`` "OutOfAfrica_3G09". 
+* ``description``: A brief one-line description of the demographic model.
+* ``long_description``: A longer description (say, a concise paragraph) that describes
+  the model in more detail.
+* ``populations``: A list of ``stdpopsim.Population`` objects, which have their own
+  ``id`` and ``description``. For example, the Thousand Genomes Project Yoruba panel
+  could be defined as ``stdpopsim.Population(id="YRI", description="1000 Genomes YRI (Yorubans)")``.
+* ``citations``: A list of ``stdpopsim.Citation`` objects for the appropriate citation
+  for this model. The citation object requires author, year, and doi information, and
+  a specified reason for citing this model.
+* ``generation_time``: The generation time for this species in years.
+
+If this is your first time implementing a demographic model in `stdpopsim`, it's a good
+idea to take some time browsing the Catalog and species' demographic models in the
+source code to see how existing models are typically written and documented. If you have
+any questions or confusion about formatting or implementing demographic models, please
+don't hesitate to open an `issue <http://github.com/popgensims/stdpopsim/issues>`_ --
+we're more than happy to answer any questions and help get you up and running.
+
+-----------------------------------
+What models are appropriate to add?
+-----------------------------------
+
+`Stdpopsim` supports any demographic model from the published literature that gives
+enough information to be able to define `msprime` demography objects. At a minimum, that
+includes population sizes and the timing of demographic events. These values need to
+either be given in "physical" units (that is, raw population sizes and time units in
+generations), or be able to be converted to physical units using, e.g., mutation rates
+used in the published study.
+
+Note that it is not necessary that the demographic model is attached to a particular
+species. `Stdpopsim` contains a collection of generic models that are widely used in
+developing and testing inference methods. If there is a generic model that does not
+currently exist in our catalog but would be useful to include, we also welcome those
+contributions. Again, there should also be a citation given for generic models.
+
+---------------------------------------
+So the model is implemented. What next?
+---------------------------------------
+
+Once you have implemented the demographic model locally, including basic unit tests and
+documentation, the next step is to open a pull request with this addition. Now at this
+point, most of your work is done!  The model is reviewed and verified by an independent
+member of the development team, and there may be some discussion about formatting and
```

Comment:
link to the demographic model review process explictly rather than saying "next section". Sections get added nad moved around.

You'll need to add an anchor name - see other parts for how this is done.